src/requirements/data/majors/oldIsst.ts

```diff
@@ -0,0 +1,391 @@
+    checker: includesWithSubRequirements(['ORIE 3300'], ['ORIE 3500']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1],
+  },
+  {
+    name: 'Information Systems',
+    description: 'INFO 2300, ORIE 3800 and INFO 3300 or INFO 4300',
+    source:
+      'https://infosci.cornell.edu/undergraduate/info-sci-majors/bs-information-science-systems-and-technology/old-isst-requirements',
+    checker: includesWithSubRequirements(['INFO 2300'], ['ORIE 3800'], ['INFO 3300', 'INFO 4300']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1, 1],
+  },
+  {
+    name: 'Economic, Organization, and Social Context',
+    description: 'INFO 2040 and INFO 2450 or ENGRC 3350',
+    source:
+      'https://infosci.cornell.edu/undergraduate/info-sci-majors/bs-information-science-systems-and-technology/old-isst-requirements',
+    checker: includesWithSubRequirements(['INFO 2040'], ['INFO 2450', 'ENGRC 3350']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1],
+  },
+  {
+    name: 'Engineering Mathematics',
+    description: 'MATH 1910, MATH 1920, MATH 2940, and MATH 2930 or MATH 3040 or CS 2800.',
+    source:
+      'https://infosci.cornell.edu/undergraduate/info-sci-majors/bs-information-science-systems-and-technology/old-isst-requirements',
+    checker: includesWithSubRequirements(
+      ['MATH 1910'],
+      ['MATH 1920'],
+      ['MATH 2940'],
+      ['MATH 2930', 'MATH 3040', 'CS 2800']
+    ),
+    allowCourseDoubleCounting: true,
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1, 1, 1],
+  },
+  {
+    name: 'Engineering Distributions',
+    description: 'ENGRD 2700 and ENGRD 2100',
+    source:
+      'https://infosci.cornell.edu/undergraduate/info-sci-majors/bs-information-science-systems-and-technology/old-isst-requirements',
+    checker: includesWithSubRequirements(['ENGRD 2110'], ['ENGRD 2700']),
+    allowCourseDoubleCounting: true,
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1],
+  },
+  {
+    // TODO: Can't accurately track C (electives are all from one category) as this is nested toggles
+    name: 'Specialization',
+    description:
+      'Specialize in either the Management Science option or the Information Science option (7 advanced courses); and by taking 2 major-approved courses',
+    source: 'http://courses.cornell.edu/preview_program.php?catoid=36&poid=17535',
+    fulfilledBy: 'toggleable',
+    fulfillmentOptions: {
+      'Information Science': {
+        description:
+          'A. Three courses from Information Systems ' +
+          'B. One course from Mathematical Modeling in IT ' +
+          'C. Three electives, all from one of Human-Centered Systems or Social Systems ' +
+          'D. Two major-approved electives chosen from areas I-VI. INFO 4900 may be used to fulfill one major-approved elective ',
+        counting: 'courses',
+        checker: includesWithSubRequirements(
```

Comment:
@SamChou19815 Is there a better way to do checkers for toggleable requirements with multiple requirements? Is using includesWithSubRequirements the only option currently? 
[A lot of these requirements are repeated so I can make them into arrays to make it easier to check but wasn't sure if I could do that]

Reply:
right now that is the only way.